### PR TITLE
Feature/29 namespaces

### PR DESCRIPTION
**Pull request by TimPurdum**
This is changing all file names, folder names, and text references from `dymaptic.Blazor.GIS.API.Core` to `dymaptic.GeoBlazor.Core`. Note that we definitely need to increment the version along with this, I'll push that in the morning, but just getting this PR started.

### CODE REVIEW

**Review by seahro — APPROVED** [review]
The Blazor.GIS.API changes to GeoBlazor look good.  I see there were a couple other adjustments that I saw were refactorings.  They seemed like simple adjustments.  Looks good

File: samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/Widgets.razor [hunk 1]
```diff
@@ -3,24 +3,27 @@
...
+        </div>
+        <div class="form-group">
+            <label>Basemap Gallery: <input type="checkbox" @onchange="@(() => ToggleWidget(nameof(BasemapGalleryWidget)))"></label>
+        </div>
+        <div class="form-group">
+            <label>Legend: <input type="checkbox" @onchange="@(() => ToggleWidget(nameof(LegendWidget)))"></label>
+        </div>
+        <div class="form-group">
+            <label>Scale Bar: <input type="checkbox" @onchange="@(() => ToggleWidget(nameof(ScaleBarWidget)))"></label>
+        </div>
```
[the remark this answers is not in the document]
Comment (TimPurdum) [reply samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/Widgets.razor:25]: Thanks for catching/pointing this out. It's actually local changes that belong on [#17](https://github.com/dymaptic/dy-blazor-api-core/issues/17). I'll revert them here.

**Comment by TimPurdum** [comment]
@seahro I must have accidentally based this branch off `feature/17_free_the_widgets` instead of off `develop`. I have undone those changes, so everything in here now should be just namespace related.

**Comment by TimPurdum** [comment]
Well, undoing those changes here made the merge in from that branch even messier. But, I think I've got it all squared away, and I have re-tested, both manually and with the playwright tests.

**Pull request merged**